heareval/predictions/runner.py

```diff
@@ -23,6 +23,13 @@
     nargs=-1,
     required=True,
 )
+@click.option(
+    "--outdir",
```

Comment:
Can we call this --predictions-dir ? Or similar naming convention as embed which uses --embeddings-dir as the output dir?